blob/azureblob/azureblob_test.go

```diff
@@ -264,113 +261,3 @@ func TestOpenBucket(t *testing.T) {
 		})
 	}
 }
-
-func TestOpenURL(t *testing.T) {
```

Comment:
Why did you remove this entirely? It looks like test coverage on Travis went way down as a result. I think you can basically put it back, just removing the tests that were focused on `cred_path`.